src/schema/users.ts

```diff
@@ -1158,25 +1170,36 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       { id }: { id: string },
       ctx: Context,
     ): Promise<GQLUserStats | null> => {
-      const [postStats, commentStats] = await Promise.all([
-        getAuthorPostStats(ctx.con, id),
-        await ctx.con
-          .createQueryBuilder()
-          .select('count(*)', 'numComments')
-          .addSelect('sum(comment.upvotes)', 'numCommentUpvotes')
-          .from(Comment, 'comment')
-          .where({ userId: id })
-          .innerJoin(Post, 'p', `comment.postId = p.id`)
-          .andWhere('p.visible = true')
-          .andWhere('p.deleted = false')
-          .getRawOne(),
-      ]);
+      const [postStats, commentStats, numFollowing, numFollowers] =
+        await Promise.all([
```

Comment:
I guess at one stage this would make a good `user_stats` mv?

Reply:
Yeah probably.